newbase/NFmiDictionaryFunction.h

```diff
@@ -0,0 +1,16 @@
+
+/*!  NFmiDictionaryFunction.h
+ * Tämä on Editorin käyttämä sana kirja funktio.
+ * Kieli versiot stringeihin tulevat täältä.
+ */
+
+#ifndef NFMIDICTIONARYFUNCTION_H
+#define NFMIDICTIONARYFUNCTION_H
+
+#include <NFmiSettings.h>
```

Comment:
Prepocessor may not find the file if angle-bracket form is used. Use quoted form instead of angle-brackets